src/Uno.UI/UI/Xaml/Controls/TimePicker/TimePickerFlyoutPopupPanel.iOS.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Windows.Foundation;
+using Windows.UI.Xaml.Controls.Primitives;
+
+namespace Windows.UI.Xaml.Controls
+{
+	internal partial class TimePickerFlyoutPopupPanel : FlyoutBasePopupPanel
```

Comment:
Isn't this the exact same as `DatePickerFlyoutPopupPanel`? Why do you have two separate classes?